client/src/Components/HomePage/postDetails2.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import more_icon from '../../images/more_icon.svg'
+import './post2.css'
+class PostDetailsTop extends React.Component{
+    constructor(props){
+        super(props)
+        this.state = {
+            name:props.name,
+            location:props.location
+        }
+    }
+    render(){
+        return <div>
+            <div> 
+                <img className='more_icon' src = {more_icon} />
```

Comment:
you dont need to give a className to this image you can style it without using it anyway its okay to do it this way.